helium-wallet/src/cmd/mod.rs

```diff
@@ -137,13 +136,9 @@ impl CommitOpts {
 
         let versioned_tx = tx.into();
         if self.commit {
-            let config = RpcSendTransactionConfig {
-                skip_preflight: self.skip_preflight,
-                ..Default::default()
-            };
             client
                 .as_ref()
-                .send_transaction_with_config(&versioned_tx, config)
+                .send_and_confirm_transaction(&versioned_tx)
```

Comment:
see above.. the workaround for this would be the same as doing: https://solana.stackexchange.com/questions/12506/how-to-use-sendandconfirmtransaction-with-versionedtransaction ?

Reply:
I thought I mentioned the pre-flight problem on Discord, so that's why I thought you were ok with it.

>  The only downside I can see is that this new method doesn't appear to have any "pre-flight" checks that send does.

I guess we'll have to research whether there's an issue with VersionedTransaction and the Rust crate or if it's purely a problem with the web3.js version alone.